doc/build_doc/linux/README.md

```diff
@@ -28,15 +31,26 @@ cd ..
 source setupvars.sh
 ```
 * OpenVINO Tokenizers Extension - Download package from [here](https://storage.openvinotoolkit.org/repositories/openvino_tokenizers/packages/2024.3.0.0/). For these instructions, we will use ```openvino_tokenizers_ubuntu22_2024.3.0.0_x86_64.tar.gz```.
+Again, if don't want to click through a bunch of links and keep things on the commandline/terminal, you can use wget:
+`wget https://storage.openvinotoolkit.org/repositories/openvino_tokenizers/packages/2024.3.0.0/openvino_tokenizers_ubuntu22_2024.3.0.0_x86_64.tar.gz`
 ```
 # extract it (this will create and populate a 'runtime' folder)
 tar xzvf openvino_tokenizers_ubuntu22_2024.3.0.0_x86_64.tar.gz
 
-# copy extension libraries into OpenVINO lib folder:
-cp runtime/lib/intel64/ _openvino_toolkit_ubuntu22_2024.3.0.16041.1e3b88e4e3f_x86_64/runtime/lib/intel64/
-```
+# To copy `libcore_tokenizers.so` && `libopenvino_tokenizers.so` to the openvino toolkit directory:
+`
+cp -r ~/audacity-openvino/openvino_tokenizers/runtime/lib/intel64/* ~/audacity-openvino/openvino_toolkit/l_openvino_toolkit_ubuntu22_2024.3.0.16041.1e3b88e4e3f_x86_64/runtime/lib/intel64
```

Comment:
This cmd seems to assume that user has created ~/audacity-openvino and downloaded things there. Did you mean to do that?

Reply:
Yes and no. I included an opinionated location because I think it would be best to recommend/suggest one in the documentation, but I can definitely understand that that might not be the direction you want to take with things style-wise.

I think it would be better to give explicit directions in the structure and naming of the sub directories as things get built and put together. I'm thinking from the perspective of someone who may be new to all of this, while someone with more experience could structure things the way they want to.

I could flesh things out further if you want to take the opinionated route, or you or I could remove the directory designation.

Reply:
Got it. I think you can either replace ```~/audacity-openvino``` with ```<path_to>``` (or something like that), or somewhere at the beginning of the README you can give the reader a heads-up that you'll be downloading packages to a working directory that you've created in your home directory, ```~/audacity-openvino```, and then I think it will be clear.

Thanks again!